app/cdap/components/ConnectionList/styles.ts

```diff
@@ -15,6 +15,8 @@
  */
 
 import { makeStyles } from '@material-ui/styles';
+import { blue } from '@material-ui/core/colors';
```

Comment:
Nit: `import { blue, gray } from '@material-ui/core/colors';`

Reply:
And throughout

Reply:
Okay, fixed